script/Arcueid/Day 5/05_00_ARC05_9_1.txt

```diff
@@ -136,12 +139,12 @@ So if possible, she should throw three pieces of debris.
 [sha:64b9a05a676348ef9fa06f60c5706a805e063bbb]{
 -- Page 781, Offset 10777.
 -- 一発目はヴローヴを直接狙い、
-The first shot should be aimed directly at Vlov,
+The first shot should be aimed directly at Vlov―――
 }
 [sha:84f93710c2edd6d0fa271e2dd3e920b48d04b600]{
 -- Page 781, Offset 10778.
 -- 二発目はわざと狙いを外してヴローヴの後方に、
-the second shot should deliberately miss and land behind him,
+the second shot should deliberately miss and land behind him―――
```

Comment:
should this be capitalized? ross said he's been capitalizing lines like these but i'm not sure if this qualifies

Reply:
Nope.